src/lib/bootloader/autoyast_converter.rb

```diff
@@ -180,6 +193,17 @@ def export_stage1(res, stage1)
         res["boot_custom"] = stage1.custom_devices.join(",") unless stage1.custom_devices.empty?
       end
 
+      # only for grub2, not for others
+      GRUB2_BOOLEAN_MAPPING = {
+        "trusted_grub" => :trusted_boot
+      }
```

Comment:
I prefer to have constant definition near others and ideally before its first usage (in this case in import)